plat/kvm/arm/setup.c

```diff
@@ -208,6 +213,14 @@ static void _libkvmplat_entry2(void *arg __attribute__((unused)))
 void _libkvmplat_start(void *dtb_pointer)
 {
 	_init_dtb(dtb_pointer);
+
+#ifdef CONFIG_HAVE_SMP
+	int ret = smp_init(_libkvmplat_cfg.dtb);
+
+	UK_ASSERT(ret == 0);
```

Comment:
Hmmm, I think it would be preferably to print an error message followed by a UK_CRASH()